dbytecoderc/Questioner
# #163421750 Added logic to enable users to comment

#### What does this PR do?
> - This PR adds the functionality to enable users post a comment on a question

#### Description of Task to be completed?
> - Have the POST `/api/v1/comments` route working
#### How should this be manually tested?
> - Clone the repo and run `npm install` to install dependencies.
> - Download and install postgresql from this [link](https://www.postgresql.org/)
> - Setup postgreql using this [guide](https://www.postgresql.org/docs/10/runtime.html)
> - run `npm migrate` to create the tables
> - run `npm start:dev` to start server
> - check the `server/test/testData` folder route for the input formats and proceed to testing with postman.

#### What are the relevant pivotal tracker stories?
> - `#163421750`

#### NB
> - I have written the model logic for `rsvps` and `votes` but I have not hooked in the routes yet so you can just ignore that part and I also fixed the variables to fit the recommended `camelcase` or `snakecase` format
